tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -252,3 +238,71 @@ def _verify_built_artifact(self, build_dir, function_logical_id, expected_files,
         gem_path = ruby_bundled_path.joinpath(ruby_version[0], 'gems')
 
         self.assertTrue(any([True if self.EXPECTED_RUBY_GEM in gem else False for gem in os.listdir(str(gem_path))]))
+
+
+class TestBuildCommand_JavaGradle(BuildIntegBase):
+
+    EXPECTED_FILES_PROJECT_MANIFEST = {'aws', 'lib', "META-INF"}
+    EXPECTED_DEPENDENCIES = {'annotations-2.1.0.jar', "aws-lambda-java-core-1.1.0.jar"}
+
+    FUNCTION_LOGICAL_ID = "Function"
+    USING_GRADLE_PATH = os.path.join("Java", "gradle")
+    USING_GRADLEW_PATH = os.path.join("Java", "gradlew")
+
+    @parameterized.expand([
+        ("java8", USING_GRADLE_PATH, False),
+        ("java8", USING_GRADLEW_PATH, False),
+        # ("java8", USING_GRADLE_PATH, "use_container"),
+        # ("java8", USING_GRADLEW_PATH, "use_container"),
+    ])
+    def test_with_gradle(self, runtime, code_path, use_container):
+        overrides = {"Runtime": runtime, "CodeUri": code_path, "Handler": "aws.example.Hello::myHandler"}
+        cmdlist = self.get_command_list(use_container=use_container,
+                                        parameter_overrides=overrides)
+
+        LOG.info("Running Command: {}".format(cmdlist))
+        process = subprocess.Popen(cmdlist, cwd=self.working_dir)
+        process.wait()
+
+        self._verify_built_artifact(self.default_build_dir, self.FUNCTION_LOGICAL_ID,
```

Comment:
Can we look at built artifacts verify if we used `gradle` or `gradlew`?

Reply:
Ah good point. I don't know. I will check with Dongie